__tests__/app.test.js

```diff
@@ -0,0 +1,64 @@
+const { app, endpoints } = require("../app");
+const request = require("supertest")
+const req = request(app);
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => seed(testData)) 
+afterAll(() => db.end())
+
+describe("app.js", () => {
+  describe("/api/topics", () => {
+    test("GET:200 sends an array of topics to the client", () => {
+      return req
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.topics)).toBe(true);
+          expect(body.topics).toHaveLength(3);
+        });
+    });
+    test("GET:200 each topic has the correct properties", () => {
+      return req
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.topics.forEach((topic) => {
```

Comment:
I would test for the length of the array before using a forEach to test for multiple things